edx-platform/pearson-theme/lms/templates/header/navbar-logo-header.html

```diff
@@ -0,0 +1,44 @@
+## mako
+
+<%page expression_filter="h" args="online_help_token"/>
+
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from lms.djangoapps.ccx.overrides import get_current_ccx
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+# App that handles subdomain specific branding
+from branding import api as branding_api
+%>
+
+<h1 class="header-logo">
+  <a href="http://staging.pfn.proversity.io/">
```

Comment:
@Adonisrq94 this url here can not be locked to staging, if we merge this to production it will require a change to make this work for production, please change this to be the "LMS_ROOT_URL" from the configuration_helpers